freebsd/FreeBSDProcessList.c

```diff
@@ -494,10 +494,6 @@ void ProcessList_goThroughEntries(ProcessList* this) {
                proc->percent_cpu = proc->percent_cpu / (double) cpus;
          }
       }
-      if (isIdleProcess == false && proc->percent_cpu >= 99.8) {
-         // don't break formatting
-         proc->percent_cpu = 99.8;
-      }
```

Comment:
I'm not sure what was supposed to be happening here, but it seems misguided.

I suspect the check for `isIdleProcess` is just an optimization to avoid doing the floating point comparison when unnecessary, and the cap at 99.8 seems unnecessary as you've mentioned. The kernel will actually directly report percentages greater than 100% in the case of multiple threads, and we definitely want to show that.